AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -11,7 +11,6 @@ extension Target.Dependency {
     static var AWSSDKIdentity: Self { "AWSSDKIdentity" }
     static var AWSSDKIdentityAPI: Self { "AWSSDKIdentityAPI" }
     static var AWSSDKChecksums: Self { "AWSSDKChecksums" }
-    static var AWSSDKPartitions: Self { "AWSSDKPartitions" }
```

Comment:
The `AWSSDKPartitions` target is removed, since it is no longer used to provide partition info.